packages/blueprints-integration/src/rundown.ts

```diff
@@ -137,20 +137,26 @@ export interface IBlueprintPart<TMetadata = unknown> extends IBlueprintMutatable
 	 */
 	invalid?: boolean
 	/**
-	 * Provide additional information about the reason a part is invalid. The title should be a single, short sentence describing the reason. Additional
-	 * information can be provided in the description property. The blueprints can also provide a color hint that the UI can use when displaying the part.
+	 * Provide additional information about the reason a part is invalid. The `key` is the string key from blueprints
+	 * translations. Args will be used to replace placeholders within the translated file. If `key` is not found in the
+	 * translation, it will be interpollated using the `args` and used as the string to be displayed.
+	 * The blueprints can also provide a color hint that the UI can use when displaying the part.
 	 * Color needs to be in #xxxxxx RGB hexadecimal format.
 	 *
 	 * @type {{
-	 * 		title: string,
-	 * 		description?: string
+	 * 		message: {
+	 * 			key: string,
+	 * 			args?: { [key: string]: any }
+	 * 		},
 	 * 		color?: string
 	 * 	}}
 	 * @memberof IBlueprintPart
 	 */
 	invalidReason?: {
-		title: string
-		description?: string
+		message: {
```

Comment:
I would prefer the `message` type here to be made its own exported interface, so that inside the blueprints it can be referenced more cleanly. Currently there is a lot of `IBlueprintPart['invalidReason']` and I expect to want to reference the type of message here too

Reply:
Maybe we should just make a type for parametrized strings then? Basically what is here with a text property and optional args as a key/value object?

Reply:
That's a good idea, though I think it's going to be used in invalidReason only, I think? In other places it seems to be that the translated messages are sent through functions, and there it is just more conveniant to write warn('something'), rather than wrapping it in an object. @Julusian what do you mean by "type of message"?

Reply:
It is currently only used here, but I do want to be able to easily reference that type in the blueprints. 
I meant the `IBlueprintPart['invalidReason']['message']` type.

My motivation for this is to change the blueprints to have the `t` function:
```
export function t(key: string, args: { [key: string}: any }): ITranslatableMessage {
  return { key, args }
}
```

Reply:
If that type now happens, I'd like to propose that we don't use "key" as a property name for the message text string. It's really only a key in the context of i18n translations, in all other contexts it's the actual message text :)

Reply:
Well, it's still technically, the `key` to the translations bundle. We're using the optional, and very conveniant way of "English text is the key", but it's not exactly neccessary to use it that way.

Reply:
Ok, how about now? The blueprint integrations now have their own translatable message interface. I still think it makes sense to use `key` here, since that's what it technically is.